module/API/Measurer.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file Measurer.cpp
+ * @brief 計測に用いる関数
```

Comment:
ラッパーって書いた方がより良いかな[（参考資料）](https://qiita.com/dsudo/items/51ca822169ef9f8b9bc3)
```suggestion
 * @brief 計測に用いる関数をまとめたラッパークラス
```